README.md

```diff
@@ -16,21 +16,18 @@ feedback.
 
 This code is known to work on Go 1.8 and above. We recommend always using the newest stable release of Go.
```

Comment:
Do we need to bump supported Go version?

Reply:
This will still work with go 1.8 but you are correct that the git clone and build workflow will not work and if we want to maintain both I can add docs for 1.8 vs 1.11+.

Reply:
Yeah, if you could do that, it would be nice. Go 1.10 is the default version in Ubuntu for example, so I think it would be better to also have instructions covering Go <1.11.

Reply:
All set, I added instructions for 1.8-1.10 and 1.11+.